### DISCUSSION TITLE
Using bookmark panel after convert to bookmark panel?

### DISCUSSION THREAD

**Discussion by ewelling-csc** [mbnuqw/sidebery#1684] [Q&A]
Hi, I'm unsure about what's the idea behind "convert to bookmarks panel". It saves the tabs in a bookmark folder but after that the panel's functionality reduces to opening one bookmark and the panel and then auto converting back to tabs panel with only that one tab. What's the idea behind this?

There are also the options "save to bookmarks" and "restore from bookmarks" so I don't understand what convert to bookmarks panel is supposed to provide (or is it not supposed to be used by left clicking etc.)?

After disabling "convert to source tabs panel on opening bookmark" it starts to behave like a bookmark opener which makes some sense but I would expect some other functionality.

I'm getting the feeling I'm missing some bigger picture with this "bookmark panel" feature?

**Comment by ckeeney** [comment]
It's one year later, and I have the same question/confusion.  Is there some documentation or tutorial on how to use the Bookmarks Panel?

As soon as I click on a bookmark, it replaces the entire bookmark panel with just the single tab of the bookmark I opened.

**Comment by megamorphg** [comment]
The functionality of Save/Restore to Bookmarks is explained here:
https://github.com/mbnuqw/sidebery/discussions/1488#discussioncomment-8617442

Effectively, you can push/pull a panel with Firefox Sync!

I'm not sure what the purpose of Bookmark panels are though tbh.
